Makefile

```diff
@@ -54,7 +61,14 @@ lint-go:  # lints the Go files
 	gometalinter.v2
 
 lint-markdown: build  # lints the Markdown files
-	node_modules/.bin/prettier -l "{,!(vendor)/**/}*.md"
+	@find . -type f \( \
+		-path '**/*.md' -o \
+		-path '**/*.yml' -o \
+		-path '**/*.json' -o \
+		-path '**/*.js' \) | \
+		grep -v node_modules | \
+		grep -v vendor | \
+		xargs node_modules/.bin/prettier -l
```

Comment:
This find command probably doesn't work on vanilla Windows, but we can deal with that later.